pkgs/by-name/li/libguestfs-appliance-debian/package.nix

```diff
@@ -0,0 +1,113 @@
+{
+  lib,
+  stdenvNoCC,
+  vmTools,
+  fetchurl,
+  runCommand,
+  qemu-utils,
+}:
+let
+  debianSnapshot = "20250227T210548Z";
+
+  debianArch =
+    {
+      "x86_64-linux" = "amd64";
+      "aarch64-linux" = "arm64";
+    }
+    .${stdenvNoCC.hostPlatform.system};
+
+  debClosureForArch =
+    arch:
+    vmTools.debClosureGenerator rec {
+      name = "libguestfs-appliance-deb-closure-${arch}";
+      packages = vmTools.commonDebianPackages ++ [
+        "libguestfs-tools"
+        "linux-image-${arch}" # required by supermin
+      ];
+      packagesLists = [
+        (fetchurl {
+          url = "${urlPrefix}/dists/testing/main/binary-${arch}/Packages.xz";
+          hash =
+            {
+              "amd64" = "sha256-llPpp2k6wDAHTXFMT24ANUTIcj4SDsE16oc35la4JYk=";
+              "arm64" = "sha256-EU3tANE/SB9rM1IZw80GT5UyewpWjmfcb35HYd5cHPo=";
+            }
+            .${arch};
```

Comment:
same as below

Reply:
`arch` is one of these two (see call site of debClosureForArch)